modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -165,14 +166,48 @@ NOTE: When you create a new mountpoint using Login credentials, you must log out
 
 NOTE: Be aware that by default changes in the file system won't be transferred to ownCloud (use case: Sync client) and a **service account with access to shares** is required for ownCloud to receive changes from the personal mounts.
 
-Users create their own WND mounts on their Personal pages. These are
-created the same way as Admin-created shares. Users have four options
-for login credentials:
+Users create their own WND mounts on their Personal pages. 
+These are created the same way as Admin-created shares. 
+Users have five options for login credentials:
 
 * Username and password
 * Log-in credentials, save in session
 * Log-in credentials, save in database
 * Global credentials
+* Config-based credentials
+
+==== Config-based Credentials
+
+Config-based credentials provides a universal account for accessing an ownCloud instance.
```

Comment:
"universal account" sounds to me that it's only one account. Each user will have a different account although the password for all of them will be the same.

Reply:
Well said. I'll find another name for it.